accepted/future-releases/async-star-behavior/feature-specification.md

```diff
@@ -0,0 +1,108 @@
+# Behavior of `yield`/`yield*` in `async*` functions and `await for` loops.
+
+**Author**: [lrn@google.com](mailto:lrn@google.com)
+
+**Version**: 1.0 (2018-11-30)
+
+## Background.
+
+The Dart language specification defines the behavior of `async*` functions,
+and `yield` and `yield*` statements in those, as well as the behavior of
+`await for` loops.
+
+This specification has not always been precise, and the implemented behavior
+has been causing user problems ([https://github.com/dart-lang/sdk/issues/34775],
```

Comment:
These links are hard to ready, can you do like `[#34775](https://github.com/dart-lang/sdk/issues/34775)`

Reply:
Fone